// [[Reason: Requires adding a type reference directive that only TSC can detect]] ////

//// [tests/cases/compiler/typeReferenceDirectives8.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -7,9 +7,8 @@
 export declare const bar: invalid;
 
 /// [Errors] ////
 
-/mod1.ts(1,24): error TS9024: Declaration emit for this file requires adding a type reference directive which are not supported with --isolatedDeclarations
 /mod2.ts(2,20): error TS9010: Variable must have an explicit type annotation with --isolatedDeclarations.
 
 
 ==== /mod2.ts (1 errors) ====
@@ -20,9 +19,7 @@
 !!! related TS9027 /mod2.ts:2:14: Add a type annotation to the variable bar.
 ==== /types/lib/index.d.ts (0 errors) ====
     interface Lib { x }
     
-==== /mod1.ts (1 errors) ====
+==== /mod1.ts (0 errors) ====
     export function foo(): Lib { return {x: 1} }
-                           ~~~
-!!! error TS9024: Declaration emit for this file requires adding a type reference directive which are not supported with --isolatedDeclarations
     
\ No newline at end of file
